lib/test.js

```diff
@@ -30,11 +30,15 @@ var commands = async (function(commandarr,catalog,exit_if_fail) {
 				return false;
 			}
 		}
-		commandarr[i].testnumber = i;
-		commandarr[i].catalog = catalog;
-		return testoutput(commandarr[i], out.toString());
+
```

Comment:
There was an error associated with [commander](https://github.com/hapi-server/server-nodejs/blob/master/lib/metadata.js#L537) object received to the [commands](https://github.com/hapi-server/server-nodejs/blob/master/lib/test.js#L7) function. It did not have [test_number](https://github.com/hapi-server/server-nodejs/blob/master/lib/test.js#L24) and [catalog](https://github.com/hapi-server/server-nodejs/blob/master/lib/test.js#L25) types associated with it. So, created a new object with necessary attributes to return.